### PR TITLE
feat(readme): update readMe, update replace method

### PR DESCRIPTION
**Pull request by denStrigo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,9 +96,58 @@ We recommend using SVG icons due to better rendering and performance capabilitie
...
+| animation | object | none    | [Icon animation](#animation)  |
+
+### Animation 
+- Add the `data-eva-animation` attribute with the animation type `(zoom, pulse, shake and flip)` to an element:
+
+```html
+<i data-eva="github" data-eva-animation="zoom"></i>
+```
+
+- By default animation works by hover and has infinity `false`, if you want change this behavior add `data-eva-hover="true | false"` and `data-eva-infinite="true | false"`
```
Comment (nnixaa) [inline README.md]: this sentence is quite hard to understand, try to rephrase

File: README.md [hunk 2]
```diff
@@ -86,9 +96,58 @@ We recommend using SVG icons due to better rendering and performance capabilitie
...
+  aniamtion: {
+    type: string, // zoom, pulse, shake, flip
+    hover: booleam, // default true
+    infinite: boolean, // default false
+  }
+});
+```
+> **Note:** The animation will be applied to all replaced elements.
+
+- Add class `eva-parent-hover` to parent container if you want that the animation works by hover on the parent element.
```
Comment (nnixaa) [inline README.md]:
```suggestion
- Add `eva-parent-hover` class to the parent container in a case you want to activate the animation hovering on the parent element.
```

File: README.md [hunk 3]
```diff
@@ -86,9 +96,60 @@ We recommend using SVG icons due to better rendering and performance capabilitie
...
+
+### Animation 
+- Add the `data-eva-animation` attribute with the animation type `(zoom, pulse, shake and flip)` to an element:
+
+```html
+<i data-eva="github" data-eva-animation="zoom"></i>
+```
+
+- Additional animation attributes:
+  * `data-eva-hover`: Makes an animation available by hover. Default value is `true`. Available true or false.
```
Comment (nnixaa) [inline README.md]:
```suggestion
  * `data-eva-hover`: Makes the animation available on hover. Default value is `true`. Available `true` or `false`.
```

File: README.md [hunk 4]
```diff
@@ -86,9 +96,60 @@ We recommend using SVG icons due to better rendering and performance capabilitie
...
+  * `data-eva-infinite`: Makes animation as infinity. Default value is `false`. Available true or false.
```
Comment (nnixaa) [inline README.md]:
```suggestion
  * `data-eva-infinite`: Makes the animation infinite. Default value is `false`. Available `true` or `false`.
```

Comment (denStrigo) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (denStrigo) [reply README.md]: done

**Pull request merged**